### PR TITLE
Split administer media permission

### PR DESCRIPTION
**Pull request by webflo**
The "administer media" is very powerful atm. I think we should split the permission in two similar to node module.

### CODE REVIEW

File: src/Tests/MediaUITest.php [hunk 1]
```diff
@@ -40,9 +40,7 @@ protected function setUp() {
 
     $this->adminUser = $this->drupalCreateUser(array(
       'administer media',
-      'administer media fields',
```
Comment (webflo) [inline src/Tests/MediaUITest.php:43]: Looks like this was the original idea :)

**Comment by davereid** [comment]
Shouldn't it be named 'administer media entities' otherwise there could be a potential conflict with Media.module's permissions for D8?

**Comment by webflo** [comment]
The same rule applies to view media, update media et all. 

@slashrsm You opinion?

**Comment by slashrsm** [comment]
Sure, we can do it the way @davereid suggested.

**Comment by slashrsm** [comment]
Rebased and renamed permission to "administer media bundles" to better align with the terminology that we're using. Commit: ae89cf4

This was not touching "administer media" permission. Let's rename that in a separate issue: https://www.drupal.org/node/2716563

**Pull request closed without merging**